client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -455,6 +467,97 @@ class AdsFormSettings extends Component {
 		);
 	}
 
+	privacy() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<FormSectionHeading>{ translate( 'Privacy and Consent' ) }</FormSectionHeading>
+				<FormFieldset>
+					<SupportInfo
+						text={ translate(
+							'Enables a targeted advertising opt-out link for California consumers, as required by the California Consumer Privacy Act (CCPA).'
+						) }
+						link="https://wordpress.com/support/your-wordpress-com-site-and-the-ccpa/"
+					/>
+					<CompactFormToggle
+						checked={ !! this.state.ccpa_enabled }
+						disabled={ this.props.isLoading }
+						onChange={ this.handleCompactToggle( 'ccpa_enabled' ) }
+					>
+						{ translate( 'Enable targeted advertising to California site visitors (CCPA)' ) }
+					</CompactFormToggle>
+
+					<div className="ads__child-settings">
+						<FormSettingExplanation>
+							{ translate(
+								'For more information about the California Consumer Privacy Act (CCPA) and how it pertains to your site, please consult our {{a}}CCPA guide for site owners{{/a}}.',
+								{
+									components: {
+										a: (
+											<a
+												href="https://wordpress.com/support/your-wordpress-com-site-and-the-ccpa/"
+												target="_blank"
+												rel="noopener noreferrer"
+											/>
+										),
+									},
+								}
+							) }
+						</FormSettingExplanation>
+					</div>
+				</FormFieldset>
+
+				{ this.state.ccpa_enabled && (
+					<div className="ads__child-settings">
+						<FormFieldset>
+							<FormLabel>{ translate( 'Do Not Sell Link' ) }</FormLabel>
+							<span>
+								{ translate(
+									'CCPA requires that you place a "Do Not Sell My Personal Information" link on every page of your site where targeted advertising will appear. You can use the {{a}}Do Not Sell Link (CCPA) Widget{{/a}}, or the {{code}}[ccpa-do-not-sell-link]{{/code}} shortcode to automatically place this link on your site. Note: the link will always display to logged in administrators regardless of geolocation.',
+									{
+										components: {
+											a: (
+												<a
+													href={ this.props.widgetsUrl }
+													target="_blank"
+													rel="noopener noreferrer"
+												/>
+											),
+											code: <code />,
+										},
+									}
+								) }
+							</span>
+							<FormSettingExplanation>
+								{ translate( 'Failure to add this link will result in non-compliance with CCPA.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+
+						<FormFieldset>
+							<FormLabel htmlFor="ccpa-privacy-policy-url">
+								{ translate( 'Privacy Policy URL' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **93** times:
`translate( 'privacy-policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wordads-ccpa)